tasks/eth/029-holocene-system-config-upgrade-and-init-base/NestedSignFromJson.s.sol

```diff
@@ -34,6 +34,10 @@ contract NestedSignFromJson is OriginalNestedSignFromJson, CouncilFoundationNest
         vm.assertEq(systemConfig.eip1559Denominator(), 250, "incorrect EIP1559 denominator");
         vm.assertEq(systemConfig.eip1559Elasticity(), 2, "incorrect EIP1559 elasticity");
 
+        vm.assertEq(systemConfig.blobbasefeeScalar(), 1055762, "incorrect blobbasefeeScalar");
+        vm.assertEq(systemConfig.basefeeScalar(), 2269, "incorrect basefeeScalar");
+        vm.assertEq(systemConfig.gasLimit(), 96000000, "incorrect gasLimit");
```

Comment:
How do we know these are the desired values? I think there should be a bullet in the README attesting to these

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/513/commits/804676f43fc6d354e4266afac60f33e7d7d3999e